qemu/tests/iozone_linux.py

```diff
@@ -0,0 +1,29 @@
+from virttest import error_context
+from provider.storage_benchmark import generate_instance
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run IOzone for linux on a linux guest:
+    1) Log into a guest.
+    2) Execute the IOzone test.
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    iozone_test_dir = params.get('iozone_test_dir', '/home')
+    iozone_cmd_options = params['iozone_cmd_options']
+    iozone_timeout = float(params.get("iozone_timeout", 18000))
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=float(params.get("login_timeout", 360)))
+    error_context.context("Running IOzone command on guest.")
+    try:
+        iozone = generate_instance(params, session, 'iozone')
+        iozone.run(iozone_cmd_options % iozone_test_dir, iozone_timeout)
```

Comment:
iozone_test_dir will always '/home' here. with_system_disk test, it's ok, but for with_data_disk test,  it should be data disk mounted path, run for /home, that's remain test for system disk.
or you can confirm with xuwei or coli if they want to cover with_data_disk for this case~

Reply:
thanks, I have updated,  it's my fault. I forgot it.